src/components/pages/Categories/List.vue

```diff
@@ -66,6 +78,18 @@ export default {
         this.$store.dispatch('categories/getAllCategories');
       });
     },
+    openModal(categoryId, categoryName) {
+      const deleteCategory = {
+        id: categoryId,
+        name: categoryName,
+      };
+      this.$store.dispatch('categories/confirmDeleteCategory', deleteCategory);
```

Comment:
[nits]

一時変数は、逆に可読性を落としてしまいます。

```suggestion
      this.$store.dispatch('categories/confirmDeleteCategory', {
        id: categoryId,
        name: categoryName,
      });
```

Reply:
変更しておきました。その場限りの変数なら、変数に代入→変数を使う、より値を直接入力してしまった方がいいんですね。